src/sql/base/browser/ui/selectBox/selectBox.ts

```diff
@@ -134,8 +134,7 @@ export class SelectBox extends vsSelectBox {
 	}
 
 	public onSelect(newInput: ISelectData) {
-		const selected = this._dialogOptions[newInput.index];
-		this._selectedOption = selected.value;
+		this.select(newInput.index);
```

Comment:
Pretty sure we don't want to do this - onSelect is called when the base onDidSelect is fired which we already hook in to to fire our own onDidSelect. Calling this.select here will result in two onDidSelect being fired. 

Why did you make this change anyways?